src/components/UserProfile/SideWidget/SideWidget.js

```diff
@@ -1,33 +1,46 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import BarChart from 'react-feather/dist/icons/bar-chart-2';
+import BookOpen from 'react-feather/dist/icons/book-open';
 
 import SideWidgetLink from './SideWidgetLinks/SideWidgetLink';
 import './SideWidget.scss';
 
-const profileLinks = [
-  {
-    id: 1,
-    class: 'side-card__link',
-    title: 'Articles',
-    imageUrl: '',
-    path: '/my-articles',
-  },
-  {
-    id: 2,
-    class: '',
-    titie: 'Stat',
-    imageUrl: '',
-    path: '/profile-stat',
-  },
-];
+const SideWidget = (props) => {
+  const profileLinks = [
```

Comment:
Hi @JohnMadakin. LOVE THIS!. But i think it would be better if you move this `profileLinks` to aanother file. Say we have a lot of `profileLinks` what then happens

Reply:
done